gbfs.md

```diff
@@ -396,7 +397,7 @@ The following fields are all attributes within the main `data` object for this f
 Field Name | REQUIRED | Type | Defines
 ---|---|---|---
 `system_id` | Yes | ID | This is a globally unique identifier for the vehicle share system. Each distinct system or geographic area in which vehicles are operated MUST have its own unique `system_id`. It is up to the publisher of the feed to guarantee uniqueness and MUST be checked against existing `system_id` fields in  [systems.csv](https://github.com/MobilityData/gbfs/blob/master/systems.csv) to ensure this. This value is intended to remain the same over the life of the system. <br><br> System IDs SHOULD be recognizable as belonging to a particular system as opposed to random strings - for example, `bcycle_austin` or `biketown_pdx`.
-`languages` <br/>*(added of v3.0-RC)* | Yes | Array | List of languages used in translated strings. Each element in the list must be of type Language.
+`languages` <br/>*(added of v3.0-RC)* | Yes | Array<Language> | List of languages used in translated strings.
```

Comment:
I think that making the type contained in the arrays is a good idea (eg: `Array<Object>`, `Array<Language>`, etc) 👍

Reply:
I think this could be more (in this PR or in a follow-up one), I didn't do it yet to make this objective of this PR more visible.